src/PIL/ImageCms.py

```diff
@@ -32,7 +34,7 @@
 
     _imagingcms = DeferredError.new(ex)
 
-DESCRIPTION = """
+_DESCRIPTION = """
```

Comment:
When removing this constant, is there any important info in this text we should keep somewhere?

Reply:
> Originally released under LGPL.  Graciously donated to PIL in
    March 2009, for distribution under the standard PIL license

This should probably be turned into a comment - I've pushed a commit.

There are also links to https://www.cazabon.com/pyCMS and https://www.littlecms.com. I'm not sure these need to be kept; https://www.cazabon.com/pyCMS is the first result I get on Google for "Kevin Cazabon's PyCMS library" in a private window.

Otherwise, it just duplicates information from comments in `ImageCms.py`, text from `ImageCms.rst`, and part of `CHANGES.rst` for PIL 1.1.7c1.